.github/ISSUE_TEMPLATE/2-feature-request.yml

```diff
@@ -0,0 +1,65 @@
+name: Feature Request
+description: Use this template requesting new functionality. Please keep it reasonable :)
+labels: feature, needs triage
+body:  
+  - type: markdown
+    attributes:
+      value: |
+        # Feature Request
+        Thank you for opening a feature Request! Please take the time to properly and thoughtfully fill
+        out this form to make it easier for us to work with it
+       
+        If you notice you cant fully express your feature in this template, you can always open a
+        blank template with all the creative freedom you'll need
+        
+        ⚠️ One more Info before you start filling everything out. ⚠️
+        If you are *not* using the Beta/Redesign already please do **not** open a feature request. New features
+        will only be added to the Redesign version of the app
+  
+  - type: checkboxes
+    id: type
+    attributes:
+      label: Type of Feature
+      description: You can select multiple if applicable
+      options:
+        - label: Something new
+        - label: Extension of a feature
+        - label: Improvement of a feature
+        - label: Accessibility
+        - label: None of the above
+  
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Please describe in detail what the feature is
+  
+  - type: textarea
+    id: reason
+    attributes:
+      label: Reasoning
+      description: Please describe why this feature would benefit Finamp
```

Comment:
This should be more about their use case. Which should also help to figure out how useful this would be to others, I guess